src/panels/config/script/ha-script-editor.js

```diff
@@ -100,7 +100,10 @@ class HaScriptEditor extends LocalizeMixin(NavigateMixin(PolymerElement)) {
             <ha-paper-icon-button-arrow-prev
               on-click="backTapped"
             ></ha-paper-icon-button-arrow-prev>
-            <div main-title>Script [[computeName(script)]]</div>
+            <div main-title>
+              [[localize('ui.panel.config.script.caption')]]
+              [[computeName(script)]]
```

Comment:
Let's give `computeName(script)` as variable to lokalize, so the order can be changed by the translator:

```js
[[localize('ui.panel.config.script.caption'), 'name', computeName(script)]]
```

The string in `en.json`would be:
`Script {name}`